awscrt/auth.py

```diff
@@ -401,9 +404,13 @@ class AwsSigningConfig(NativeResource):
             computed from the signable.
 
         credentials_provider (AwsCredentialsProvider): Credentials provider
-            to fetch signing credentials with.
+            to fetch signing credentials with. If the signing algorithm is
+            V4_ASYMMETRIC, ecc-based credentials will be derived from the
+            fetched credentials.
 
-        region (str): The region to sign against.
+        region (str): If signing algorithm is V4, the region to sign against.
+            If signing algorithm is V4_ASYMMETRIC, the value of the
+            X-amzn-region-set header (added in signing).
```

Comment:
trivial: quotes around "X-amzn-region-set" we do that when referencing other headers/query-params